tools/fuzzer/expr_gen.h

```diff
@@ -61,6 +105,9 @@ struct GenConfig {
 
   float parenthesize_prob = 0.5f;
```

Comment:
Side note, we should probably decrease this


![4ir6zr](https://user-images.githubusercontent.com/241450/96285381-2bafd780-0fdf-11eb-94f4-5cdb8646b9fb.jpg)